sys/net/gnrc/netif/gnrc_netif.c

```diff
@@ -1287,10 +1287,12 @@ int gnrc_netif_ipv6_add_prefix(gnrc_netif_t *netif,
         IS_ACTIVE(CONFIG_GNRC_IPV6_NIB_MULTIHOP_P6C) &&
         gnrc_netif_is_6ln(netif)) {
 
-        /* configure compression context */
-        if (gnrc_sixlowpan_ctx_update_6ctx(pfx, pfx_len, valid)) {
-            DEBUG("gnrc_netif: add compression context for prefix %s/%u\n",
-                   ipv6_addr_to_str(addr_str, pfx, sizeof(addr_str)), pfx_len);
+        if (IS_ACTIVE(CONFIG_GNRC_NETIF_IPV6_AUTO_6CTX)) {
```

Comment:
`CONFIG_GNRC_NETIF_IPV6_AUTO_6CTX` also needs to be defined for the !Kconfig case

Reply:
Done, also changed the name slightly